addons/editor/initKeybinds.sqf

```diff
@@ -75,6 +75,12 @@
     };
 }, {}, [DIK_F, [false, true, true]]] call CBA_fnc_addKeybind; // Default: CTRL + ALT + F
 
+[ELSTRING(main,DisplayName), QGVAR(curatorMoveCamTo), ["str_usract_curator_move_cam", "str_controls_tooltips_curator_move_cam"], {
+    if (!isNull curatorCamera) then {
+        [] call FUNC(handleCuratorMoveCamTo)
+    };
+}, {}, [DIK_F, [false, false, false]]] call CBA_fnc_addKeybind; // Default: F
```

Comment:
I dislike adding a separate keybind. Let's use the KeyDown EH to override the vanilla keybind. If we want to make this optional, could add a setting like "Move Camera To Behaviour" - maybe 3 options: default, CQB mode, and CQB + alternative far view.

Reply:
ah ok. Should this code go in `camera` instead of `editor`?

Reply:
I think `editor`.

Reply:
Does the setting belong under camera?

Reply:
Good point. Normally, all the keybind handling is in `editor` component but I guess with the setting, it would make more sense to be under `camera`. Not really sure what option is the best.

Reply:
For now I've implemented everything in `editor`, with the setting using the `camera` category.